### PR TITLE
DOCS-11656 - added requested changes

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

**Review by danielchan9999 — DISMISSED** [review]

**Review by schinthakindims — CHANGES_REQUESTED** [review]
one concern with this documentation is that it makes it look like distributed locking is OSv2's concern  which is not true.
We should say something like,
 "Since distributed locking is not supported in CH 1.0, any serialization to the distributed resources including OSv2 should be done by the client applications"

**Comment by schinthakindims** [comment]
Also, let's make sure this documentation is inline with what we have agreed with @sundippatel and Ganesh as part of the distributed locking KB and documentation efforts. Since this is a sensitive area, it might make sense to have a consistent communications on this issue.

File: modules/ROOT/pages/osv2-faq.adoc [hunk 1]
```diff
@@ -46,11 +46,10 @@ Yes, Object Store v2 is in the same region as the worker where the app is initia
 
 IMPORTANT: If after the first deploy, you move the app to a different region, Object Store v2 remains in the original region to avoid data loss. Your use of Object Store v2 never moves from one region to another. When you need to deploy both an app and object store to a different region, delete the app and re-upload the app into the new region. This action loses all the data that may exist in the storage.
 
-== How are Object Store entries divided among workers?
+== Does Object Store v2 support distributed locking for key updates?
```
Comment (schinthakindims) [inline modules/ROOT/pages/osv2-faq.adoc]: @hannanelson since OSv2 technically *cannot* support distributed locking, we should probably change this to something like
 - Synchronizing access to OSv2

Comment (hannanelson) [reply modules/ROOT/pages/osv2-faq.adoc]: @schinthakindims - Since it's no longer a question, I moved it to the OSv2 intro topic.

File: modules/ROOT/pages/osv2-guide.adoc [hunk 2]
```diff
@@ -38,6 +38,11 @@ Object Store v2 uses TLS for secure transport.
 Data at rest is stored using FIPS 140-2-compliant encryption standards.
 If you require a higher level of security, encrypt sensitive data before writing it to the object store.
 
+=== Synchronize Access to Object Store v2
+
+Because CloudHub doesn't support distributed locking, errors or data discrepancies can occur if any components that use Object Store v2 attempt to update a key simultaneously.
```
Comment (evinaku) [inline modules/ROOT/pages/osv2-guide.adoc]: OSv2 can only be used in CloudHub. Seems like an irrelevant detail for the customer. Can we frame this better?
Put the focus on how the customer should deal with the lack of this capability.

Comment (evinaku) [reply modules/ROOT/pages/osv2-guide.adoc]: I like how simple and clear the previous version was. Can we have a modified version of that:
"Object Store v2 does not coordinate key updates as it does not include a distributed locking mechanism for keys. If multiple Mule components attempt to update a key simultaneously, errors or data discrepancies can occur."

Comment (schinthakindims) [reply modules/ROOT/pages/osv2-guide.adoc]: Hi,  It is incorrect to say that OSv2  *does not* include distributed locking as even if it wants to provide distributed locking, it cannot. Distributed locking is a mule interface that needs to be implemented by CH. It sends a wrong message to our customers/field and support if we say "Object Store v2 does not coordinate key updates as it does not include a distributed locking mechanism for keys."

**Review by schinthakindims — APPROVED** [review]
LGTM

**Review by valkyrie69 — APPROVED** [review]
